doc/radosgw/cloud-transition.rst

```diff
@@ -302,6 +302,18 @@ Due to API limitations there is no way to preserve original object modification
    x-amz-meta-rgwx-source-mtime: 1608546349.757100363
    x-amz-meta-rgwx-versioned-epoch: 0
 
```

Comment:
Maybe expand attrs to attributes?
And s/all the//

Reply:
Done. Thanks! Please review.